### DISCUSSION TITLE
Postgres 17 support?

### DISCUSSION THREAD

**Discussion by Owl-Tec** [dani-garcia/vaultwarden#5073] [Q&A] [answered]
Hi Team,

Quick question - does Vaultwarden support Postgres 17 (assuming you are running a separate Postgres DB)? 

Thanks,
Calvin

**Comment by BlackDex** [comment, marked as answer]
Vaultwarden is built with the PostgreSQL v15 client library. And as far as i know the server is backwards compatible with these clients.

**Reply by Owl-Tec** [reply]
Thanks, I thought I would ask. I will update this thread when I migrate later (I am currently on Postgres 16).

**Reply by cvanfleteren** [reply]
I'm running Vaultwarden against v17 since a few days, haven't noticed any issues with it.
